### PR TITLE
logcat: add Italian translation

### PR DESCRIPTION
**Pull request by iambarge**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.it/android/logcat.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# logcat
+
+> Scarica il registro dei messaggi di sistema, comprese le _stack traces_ quando si verifica un errore, e i messaggi di log delle applicazioni.
+> Informazioni aggiuntive: <https://developer.android.com/studio/command-line/logcat>.
```
Comment (kbdharun) [inline pages.it/android/logcat.md:4]:
```suggestion
> Maggiori informazioni: <https://developer.android.com/studio/command-line/logcat>.
```

File: pages.it/android/logcat.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# logcat
+
+> Scarica il registro dei messaggi di sistema, comprese le _stack traces_ quando si verifica un errore, e i messaggi di log delle applicazioni.
```
Comment (kbdharun) [inline pages.it/android/logcat.md:3]:
```suggestion
> Scarica il registro dei messaggi di sistema, comprese le stack traces quando si verifica un errore, e i messaggi di log delle applicazioni.
```

File: pages.it/android/logcat.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`logcat`
+
+- Scrivi il log di sistema su file:
+
+`logcat -f {{percorso/al/file}}`
+
+- Mostra le righe corrispondenti ad una specifica _regular expression_:
+
+`logcat --regex {{regular_expression}}`
```
Comment (kbdharun) [inline pages.it/android/logcat.md:16]: If possible can you translate `{{regular_expression}}`?

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**